topydo/commands/ListCommand.py

```diff
@@ -69,11 +70,14 @@ def _process_flags(self):
                     self.printer = None
             elif opt == '-F':
                 self.format = value
+            elif opt == '-N':
+                # 2 lines are assumed to be taken up by printing the next prompt
+                self.limit = get_terminal_size().lines - 2
```

Comment:
Done in 9b766e9d4633055efc815c9365c6048c210294a7.

I actually set it up to print at least one item; that seemed more practical.